jmxtrans-output/jmxtrans-output-elastic/src/main/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriter.java

```diff
@@ -128,7 +128,7 @@ protected void internalWrite(Server server, Query query, ImmutableList<Result> r
 					map.put("key", values.getKey());
 					map.put("keyAlias", result.getKeyAlias());
 					map.put("value", Double.parseDouble(value.toString()));
-					map.put("timestamp", result.getEpoch());
+					map.put("timestamp", java.util.Date(result.getEpoch()));
```

Comment:
This is still changing the format of an existing field. If we have users already relying on the current behavior (and we probably have) it will break things for them. Adding a new field, while ugly, is the right way to go.

Side note: could you paste inline code instead of screenshots? That makes it much easier to search...